cli/pub_command.go

```diff
@@ -66,7 +68,7 @@ Available template functions are:
 	pub := app.Command("publish", pubHelp).Alias("pub").Action(c.publish)
 	pub.Arg("subject", "Subject to subscribe to").Required().StringVar(&c.subject)
 	pub.Arg("body", "Message body").Default("!nil!").StringVar(&c.body)
-	pub.Flag("wait", "Wait for a reply from a service").Short('w').BoolVar(&c.req)
+	pub.Flag("wait", "Wait for a single reply from a service").Short('w').BoolVar(&c.req)
```

Comment:
we should probably just remove this option entirely, think its here as a old leftover or just a mistake. pub should just pub.